### DISCUSSION TITLE
Is there a way to move a line up and down?

### DISCUSSION THREAD

**Discussion by abehidek** [helix-editor/helix#4021] [Q&A] [answered]
So I recently started using helix as a text-editor and I can't find on how can I move a line up and down, in VSCode you can use  "Alt + Up" and "Alt + Down" to move the line.

**Comment by the-mikedavis** [comment, marked as answer]
`Xdp` works for moving a single line down and `XdkP` works for moving a single line up.

There's an issue for a built-in for it: https://github.com/helix-editor/helix/issues/2245

**Reply by abehidek** [reply]
thanks, I didn't found that issue before
